cdk/lib/report.ts

```diff
@@ -17,5 +34,80 @@ export class Report extends GuStack {
 		new CfnInclude(this, 'YamlTemplate', {
 			templateFile: yamlTemplateFilePath,
 		});
+		const app = 'report';
+		const playApp = new GuPlayApp(this, {
+			access: {
+				scope: AccessScope.PUBLIC,
+			},
+			app,
+			certificateProps: {
+				domainName: props.domainName,
+				hostedZoneId: props.hostedZoneId,
+			},
+			instanceMetricGranularity: props.instanceMetricGranularity,
+			instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MICRO),
```

Comment:
Both stages use the same instance type: https://github.com/guardian/mobile-n10n/blob/58904fc970c256d576ee1893ede6e9868218c540/report/conf/report.yaml#L10-L16